documents/charter.md

```diff
@@ -50,7 +50,9 @@ This working group will not produce a solution that:
 
 2. Is appropriate for use as input to a congestion control algorithm
 
-3. Provides information other than the throughput advice 
+3. Provides information other than the throughput advice
+
+4. Signals throughput advice using off-path interfaces.
```

Comment:
So, am I reading the runes in this PR discussion correctly, that the proposed change should be something like
```suggestion
4. Signals throughput advice out of band
```
?